Automattic/jetpack
# Forms: move Single and Multiple Choice input caret to the end on focus

Fixes https://github.com/Automattic/vulcan/issues/412

## Proposed changes:

Following https://github.com/Automattic/jetpack/pull/38246, after removing a field option with the delete key, the previous item gets the focus, but the caret doesn't move to the end. This PR fixes it.

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a Jetpack test site with this branch
- Create a new post
- Add a form block
- Add the _Multiple Choice_ field
- Add several options
- Start deleting them: when a previous item gets the focus, the caret should be at the end of the text
